pkgs/development/python-modules/sqlalchemy-json/default.nix

```diff
@@ -3,18 +3,14 @@
   buildPythonPackage,
   fetchFromGitHub,
   pytestCheckHook,
-  pythonOlder,
+  setuptools,
   sqlalchemy,
 }:
 
-let
-  version = "0.7.0";
-in
-buildPythonPackage {
+buildPythonPackage rec {
```

Comment:
Just curious, is there a technical reason why `rec` would be preferable to `let-in`? The former is more explicit and has a smaller scope.

Reply:
It looks nicer, at least to me.